### DISCUSSION TITLE
can't install it

### DISCUSSION THREAD

**Discussion by qchengju** [unovue/shadcn-vue#874] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/25f632a4-f187-43df-b504-12c82e564d23)

**Comment by sadeghbarati** [comment, marked as answer]
Use `VPN` or `DNS` or `ipconfig /flushdns` then try again

**Reply by qchengju** [reply]
Thanks for your reply, but I tried it all, but it didn't work much
